Files/MultilingualResources/Files.zh-Hans.xlf

```diff
@@ -412,7 +412,7 @@
         </trans-unit>
         <trans-unit id="ModernNavigationToolbaNewTextDocument.Text" translate="yes" xml:space="preserve">
           <source>Text Document</source>
-          <target state="new">Text Document</target>
+          <target state="new">文本文档</target>

```

Comment:
The state needs to be changed to `translated` in order for the strings to show when you run the app.